src/components/profile/components/Milestones.tsx

```diff
@@ -1,121 +1,127 @@
-                    <div className="flex items-center space-x-1">
-                        <img
-                            src={twitter}
-                            alt="Tweet your activity graph"
-                            className="w-11 h-11"
-                        />
-                        <span className='text-sm font-semibold'>Tweet your activity graph</span>
-                    </div>
-                    <span className="text-purple-600 font-bold bg-white w-24 h-8 rounded-full flex items-center justify-center gap-1">
-                        +120 <span className='text-xs'>XP</span>
-                    </span>
-                </li>
-                <li className="flex items-center rounded-lg px-2 justify-between h-16 bg-gray-100">
-                    <div className="flex items-center space-x-1">
-                        <CircularProgress percentage={80} />
-                        <span className="text-sm font-semibold">Tweet your activity graph</span>
-                    </div>
-                    <span className="text-purple-600 font-bold bg-white w-24 h-8 rounded-full flex items-center justify-center gap-1">
-                        +120 <span className="text-xs">XP</span>
-                    </span>
-                </li>
-                <li className="flex items-center rounded-lg px-2 justify-between h-16 bg-gray-100">
-                    <div className="flex items-center space-x-1">
-                        <CircularProgress percentage={10} />
-                        <span className='text-sm font-semibold'>Tweet your activity graph</span>
-                    </div>
-                    <span className="text-purple-600 font-bold bg-white w-24 h-8 rounded-full flex items-center justify-center gap-1">
-                        +120 <span className='text-xs'>XP</span>
-                    </span>
-                </li>
-                <li className="flex items-center rounded-lg px-2 justify-between h-16 bg-gray-100 opacity-50">
-                    <div className="flex items-center space-x-1">
-                        <img
-                            src={twitter}
-                            alt="Tweet your activity graph"
-                            className="w-11 h-11"
-                        />
-                        <span className='text-sm font-semibold'>Tweet your activity graph</span>
-                    </div>
-                    <span className="text-purple-600 font-bold bg-white w-24 h-8 rounded-full flex items-center justify-center gap-1">
-                        +120 <span className='text-xs'>XP</span>
-                    </span>
-                </li>
-            </ul>
-        </div>
-    )
+  return (
+    <div className="bg-white p-5 rounded-xl  relative max-h-[450px]">
+      <h3 className="text-2xl mb-2 font-semibold">Milestones</h3>
+      <p className="text-sm font-inter">
+        Keep up with the team to receive rewards!
+      </p>
+      <ul className="mt-4 pb-8 space-y-4">

```

Comment:
You could have made an array and mapped over it, instead of keep repeating this code. Please do that.